examples/simple/main.tf

```diff
@@ -26,35 +26,38 @@ variable zone {
   default = "us-central1-b"
 }
 
-provider google {
-  region = "${var.region}"
+variable "service_port" {
+  default = "80"
+}
+
+variable "service_account" {
+  default = "test"
 }
 
-module "gce-lb-fr" {
```

Comment:
The external load balancer should be maintained in the example by replacing `GoogleCloudPlatform/lb/google` with `terraform-google-modules/lb/google`, `version = "~> 2.0"`.